templates/admin/src/components/features/security/SecurityBanner.tsx

```diff
@@ -0,0 +1,25 @@
+import { getGlobalInstance } from 'plume-ts-di';
+import { Alert } from '@mui/material';
+import React, { useState } from 'react';
+import { useObservable } from 'micro-observables';
+import SessionService from '../../../services/session/SessionService';
+import useMessages from '../../../i18n/hooks/messagesHook';
+
+export default function SecurityBanner() {
+  const sessionService: SessionService = getGlobalInstance(SessionService);
+  const { messages } = useMessages();
+  const hasFingerprint: boolean = useObservable(sessionService.hasFingerprint());
+
+  const [isClosed, setIsClosed] = useState(false);
+
+  return (
+    <>
+      {!isClosed && hasFingerprint && (
```

Comment:
```suggestion
      {!isClosed && !hasFingerprint && (
```
on veut pas l'afficher si on n'a pas la fingerprint plutôt ?

Reply:
Ah bah oui 🤦